contracts/modules/factories/LineFactory.sol

```diff
@@ -2,48 +2,38 @@ pragma solidity 0.8.9;
 
 import {ILineFactory} from "../../interfaces/ILineFactory.sol";
 import {IModuleFactory} from "../../interfaces/IModuleFactory.sol";
-
 import {LineLib} from "../../utils/LineLib.sol";
-
 import {SecuredLine} from "../credit/SecuredLine.sol";
```

Comment:
we can remove this import since its deployed in lib now. Can use the interface if we need to